core/src/test/java/org/infinispan/stream/DistributedStreamIteratorRepeatableReadTxTest.java

```diff
@@ -134,4 +131,25 @@ public void testKeyFilterConverterWithExistingTransaction() throws NotSupportedE
          tm.rollback();
       }
    }
+
+   public void testStreamWithMissedKeyInTransaction() throws Exception {
+      AdvancedCache<Object, String> cache = advancedCache(0, CACHE_NAME);
+      TransactionManager tm = TestingUtil.extractComponent(cache, TransactionManager.class);
```

Comment:
what is wrong with `cache.getTransactionManager()`

Reply:
I didn't think about it, I just copied the code from the method above. Let me change it everywhere.